### PR TITLE
[web wallet] Add basic react app for wallet

### PR DESCRIPTION
**Pull request by kent-white**
## Motivation
We want a basic web wallet working for easier development. This is going to move into an extension.

This PR is just to start the project. It will obviously change a ton but just wanted to get something committed so we can build on top of it. I will file some issues for the different features we want to add.

## Test Plan

yarn start

https://user-images.githubusercontent.com/19931667/164559480-82892af0-c625-482d-b99e-94a78b4add22.mov

### CODE REVIEW

File: ecosystem/web-wallet/src/pages/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    let encodedKey = Uint8Array.from(Buffer.from(key, 'hex'))
+    const account = new AptosAccount(encodedKey, undefined);
+    dispatch({account: account});
+    navigate('/wallet');
+  }
+
+  return (
+    <div className="App-header">
+      <h2>Aptos Wallet</h2>
+      <KeyInput submitKey={handleSubmitKey}/>
```
Comment (zacharydenton) [inline ecosystem/web-wallet/src/pages/Login.tsx:24]: If you use a plain `<input>` inside of a `<form>`, you will get a `submit` event when enter is pressed (you could use `<form onSubmit>` to handle it). Just make sure to `preventDefault()` the event so it doesn't reload the page. This way we won't need the `KeyInput` component (and the form will work better with screen readers).

**Comment by CapCap** [comment]
This looks good! Can we get some tests set up as well in the next diff? :-D

**Review by CapCap — APPROVED** [review]

**Comment by zacharydenton** [comment]
Looks like we need to tell cargo to ignore this directory. Add "ecosystem/web-wallet/**/*" here: https://github.com/aptos-labs/aptos-core/blob/cc5774c4584f24f8fb587bdbe8e4119d42f67eb8/x.toml#L257-L260

**Pull request merged**